frontend/templates/ng-templates/login.html

```diff
@@ -1,27 +1,57 @@
+                give you the option to update your account with the elixir information.
+            </p>
+            <center>
+                <!-- Google Login button -->
+                <a href="/google/login?next={{ login.url() }}" target="_self">
+                    <button type="button" class="btn" ng-class="{'none':'btn-primary', 'google':'btn-success', 'elixir':'btn-primary', 'elixir_google':'btn-success'}[login.loginType]">
+                        <span ng-if="login.loginType != 'google' && login.loginType != 'elixir_google'">Login with Google</span>
+                        <span ng-if="login.loginType == 'google' || login.loginType == 'elixir_google'">Logged in with Google <span class="glyphicon glyphicon-ok" aria-hidden="true"/></span>
+                    </button>
+                </a>
 
-        <!-- Elixir Login button -->
-        <a href="/elixir/login?next={{ login.url() }}" target="_self">
-            <button type="button" class="btn" ng-class="{'none':'btn-warning', 'google':'btn-primary', 'elixir':'btn-success'}[login.loginType]">Login with Elixir</button>
-        </a>
+                <!-- Elixir Login button -->
+                <a href="/elixir/login?next={{ login.url() }}" target="_self">
+                    <button type="button" class="btn" ng-class="{'none':'btn-primary', 'google':'btn-primary', 'elixir':'btn-success', 'elixir_google':'btn-success'}[login.loginType]">
+                        <span ng-if="login.loginType != 'elixir' && login.loginType != 'elixir_google'">Login with Elixir</span>
+                        <span ng-if="login.loginType == 'elixir' || login.loginType == 'elixir_google'">Logged in with Elixir <span class="glyphicon glyphicon-ok" aria-hidden="true"/></span>
+                    </button>
+                </a>
+            </center>
+            
+        </div>
 
-        <p><h4 ng-if="login.loginType == 'none'">Start by logging in using your Google account.</h4>
-           <h4 ng-if="login.loginType == 'google'">Now log in with Elixir to complete the update.</h4>
-           <h4 ng-if="login.loginType == 'elixir'">Great! Your account has been updated, you can now return and use the site as usual.</h4>
-        </p>
+    </div>
+    <div class="panel panel-info" style="width:50%; margin: 20px auto;" ng-if="login.loginType == 'elixir_google'">
+        <div class="panel-heading">Transfer Account Credentials</div>
+        <div class="panel-body">
+            <p>
+                We can now transfer your credentials from your google account to your Elixir account!
+            </p>
+            <div class="alert alert-warning">
+                Note that this will make you unable to log in with your google account in the future,
+                as your user data will be permanently updated with the elixir information.
+            </div>
+            <center>
+                <a href="/transfer?next={{ login.url() }}" target="_self">
```

Comment:
This link has to be done in a different way somehow, since the act of transfering should be a `POST`. 

Look for example how applying for a dataset is handled:
https://github.com/NBISweden/swefreq/blob/develop/frontend/src/js/factory.datasetUsers.js#L46
https://github.com/NBISweden/swefreq/blob/develop/frontend/templates/ng-templates/dataset-download.html#L54

I'm not sure how the redirect should be handled, one way is to write the js function so that it redirects on a 200 response.